### PR TITLE
Closes #2790 allow png favicon

### LINKED ISSUE #2790: PNG format for favicon
## Problem

Thank you for adding the "favicon" option, that was the only reason I used the `typedoc-plugin-extras` plugin. However it is laking the support of PNG files which is fairly standard.

## Suggested Solution

Allow .png images as well as .ico and .svg.

### PR DESCRIPTION
**Pull request by mistic100**
Solves #2790 Adds "png" to the list of allowed extensions for the favicon option

I didn't see unit tests related to this option, did I missed them ?

~~TODO : change the chinese messages~~

### CODE REVIEW

File: src/lib/internationalization/locales/en.cts [hunk 1]
```diff
@@ -388,7 +388,8 @@ export = {
         "hostedBaseUrl must start with http:// or https://",
     useHostedBaseUrlForAbsoluteLinks_requires_hostedBaseUrl:
         "The useHostedBaseUrlForAbsoluteLinks option requires that hostedBaseUrl be set",
-    favicon_must_be_ico_or_svg: "Favicon file must be either a .ico or .svg",
+    favicon_must_be_ico_or_svg:
```
Comment (Gerrit0) [inline src/lib/internationalization/locales/en.cts:391]: This is getting long enough that I think it should be made a template: `favicon_must_have_one_of_the_following_extensions_0`

**Review by Gerrit0 — CHANGES_REQUESTED** [review]
You didn't miss any unit tests, HTML output testing is all done by hand right now as I haven't found a decent way to test it which doesn't take forever to run, doesn't cost a fortune, and isn't a nightmare to maintain.

File: src/lib/internationalization/locales/en.cts (hunk 1 above)
Comment (mistic100) [reply src/lib/internationalization/locales/en.cts]: done

**Comment by Gerrit0** [comment]
Thanks!

**Pull request merged**